### ISSUE TITLE
Feature Request: flex values

### ISSUE THREAD

**Issue by tnir** [brigade/scss-lint#945]
Lint values for `flex` properties to use the predefined values.

### Bad

```css
.flex-zero {
    flex: 0;
}
.flex-zero {
    flex: 1;
}
.flex-initial {
    flex: 0 1 auto;
}
.flex-auto {
    flex: 1 1 auto;
}
.flex-none {
    flex: 0 0 auto;
}
```

### Good

```css
.flex-zero {
    flex: initial;
}
.flex-zero {
    flex: auto;
}
.flex-initial {
    flex: initial;
}
.flex-auto {
    flex: auto;
}
.flex-none {
    flex: none;
}
```

## Reference(s)
- https://developer.mozilla.org/en-US/docs/Web/CSS/flex

**Comment by sds** [comment]
Happy to accept a pull request implementing this, @tnir!
